packages/firebase_admob/README.md

```diff
@@ -184,6 +184,34 @@ RewardedVideoAd.instance.listener =
 Because `RewardedVideoAd` is a singleton object, it does not offer a `dispose`
 method.
 
+## Using native ads
+
```

Comment:
nit: May be worth to add a paragraph explicitly explaining that in order to use native adds you will have to provide Android/iOS code for displaying it.

Reply:
done https://github.com/FirebaseExtended/flutterfire/pull/169/files#diff-759637e0c287b36498a4cd4377d8a349R189